### PR TITLE
Fix AutocompleteInput and ReferenceArrayInput stories

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: packages/ra-ui-materialui/src/input/AutocompleteArrayInput.stories.tsx [hunk 1]
```diff
@@ -399,20 +399,8 @@ const BookEditWithReferenceAndCreationSupport = () => (
         }}
     >
         <SimpleForm>
-            <ReferenceArrayInput reference="authors" source="author" fullWidth>
-                <AutocompleteArrayInput
-                    create={<CreateAuthor />}
-                    options={{
```
Comment (WiXSL) [inline packages/ra-ui-materialui/src/input/AutocompleteArrayInput.stories.tsx:405]: I remove the code syntax error, but I don't know if the Idea was to render custom options here. Seems out of the scope of this story

File: packages/ra-ui-materialui/src/input/AutocompleteInput.stories.tsx [hunk 2]
```diff
@@ -403,20 +403,7 @@ const BookEditWithReferenceAndCreationSupport = () => (
     >
         <SimpleForm>
             <ReferenceInput reference="authors" source="author">
-                <AutocompleteInput
-                    create={<CreateAuthor />}
-                    options={{
```
Comment (WiXSL) [inline packages/ra-ui-materialui/src/input/AutocompleteInput.stories.tsx:404]: I remove the code syntax error, but I don't know if the Idea was to render custom options here. Seems out of the scope of this story

File: packages/ra-ui-materialui/src/input/ReferenceArrayInput.stories.tsx [hunk 3]
```diff
@@ -25,24 +27,24 @@ const dataProvider = testDataProvider({
...
+        <Form
+            onSubmit={() => {}}
+            defaultValues={{ tag_ids: [5] }}
+            render={() => (
+                <ReferenceArrayInput
+                    reference="tags"
+                    resource="posts"
+                    source="tag_ids"
+                >
+                    <DatagridInput rowClick="toggleSelection" sx={{ mt: 6 }}>
```
Comment (WiXSL) [inline packages/ra-ui-materialui/src/input/ReferenceArrayInput.stories.tsx:43]: Action buttons where hidden from view

Comment (djhi) [reply packages/ra-ui-materialui/src/input/ReferenceArrayInput.stories.tsx:43]: The DatagridInput is very WIP

**Review by djhi — APPROVED** [review]

**Pull request merged**